readme.md

```diff
@@ -198,52 +217,63 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** Describe the purpose of both morgan and body-parser. Why do you think morgan is only being run when the application is run in development mode?
 
 * **Your Answer:**
+morgan is for logging info about incoming requests. body-parser allows us to parse the response body. We only want morgan in dev mode because we wouldn't want all those logs clogging up production.
 
 ---
-
-- [ ] Update one of the `POST` routes you've created by adding the following code before the response is set. Then, try sending a request body using Postman to that route.
+- [x] Update one of the `POST` routes you've created by adding the following code before the response is set. Then, try sending a request body using Postman to that route.
   ```js
   console.log(req.body)
   ```
 
 * **Question:** Try commenting out `body-parser` in your application. What happens and why?
 
 * **Your Answer:**
-
+this throws an error because we need the body to be parsed in order for it to be returned in the application.
```

Comment:
Body parser makes it easier to process the body of a request which comes in as a data stream.